src/client/components/Sidebar.tsx

```diff
@@ -126,7 +132,7 @@ class Sidebar extends PureComponent<SidebarProps, SidebarState> {
             />
             <TimezonePicker
               currentTimezone={this.state.timezone.value}
-              onChange={timezone => this.handleChange({ timezone })}
+              onChange={timezone => this.handleTimezoneChange(timezone)}
```

Comment:
I think that this line can just be:
```onChange={this.handleTimezoneChange}```

I may be wrong so test it.

Reply:
Updated per comment. I also changed `handleTimezonChange` to an arrow function to preserve the context of `this`. Making the recommended change also improves optimization, as having the arrow function in the render means that a new function is created every time `render` is called.

Ref: https://daveceddia.com/avoid-bind-when-passing-props/